src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -187,4 +234,22 @@ public void equals() {
         assertFalse(standardCommand.equals(new EditCommand(INDEX_FIRST_PERSON, DESC_BOB)));
     }
 
+    @Test
+    public void editPersonDescriptorEquals() {
+        final EditPersonDescriptor standardDescriptor = new EditPersonDescriptor();
+        final EditPersonDescriptor otherDescriptor = new EditPersonDescriptor();
+
+        //equals to itself
+        assertTrue(standardDescriptor.equals(standardDescriptor));
+
+        //equals when the fields are the same
+        assertTrue(standardDescriptor.equals(otherDescriptor));
+
```

Comment:
Should the `not same fields is not equal` case from EditTaskCommandTest also be added here?

Reply:
sure, have added